tube/etl/outputs/es/versioning.py

```diff
@@ -99,8 +102,9 @@ def get_index_version(self, index):
                 return -1
             return 0
 
-        self.old_index_to_forget = self.es.indices.get_alias(name=index).keys()[0]
-        res = re.match('.*?([0-9]+)$', self.old_index_to_forget)
+        self.old_indices_to_forget = self.es.indices.get_alias(name=index).keys()
+        self.last_index = sorted(self.old_indices_to_forget)[-1]
```

Comment:
seems that we only work on the last one, what's happen with the others?

Reply:
The last one is to get the latest number for next alias. Others I remove in `clean_up`.